src/main/java/edu/cornell/kfs/module/purap/jaggaer/supplier/xml/AdditionalDataList.java

```diff
@@ -30,11 +30,11 @@ public void setIsChanged(String value) {
         this.isChanged = value;
     }
 
-    public List<AdditionalData> getAdditionalData() {
-        if (additionalData == null) {
-            additionalData = new ArrayList<AdditionalData>();
+    public List<AdditionalDataItem> getAdditionalDataItems() {
+        if (additionalDataItems == null) {
+            additionalDataItems = new ArrayList<AdditionalDataItem>();
         }
-        return this.additionalData;
+        return this.additionalDataItems;
```

Comment:
I am not sure we have a "Standard" convention, but if we use tools to create getters and setters for us, they include "this" and I think that is fine.  Perhaps this could be a book club topic.  I would rather defer removing "this" from the getters and setters if it turns out we need to.

Reply:
I will see about getting that clean up here